package.json

```diff
@@ -2,52 +2,58 @@
     "name": "got-scraping",
     "version": "3.2.16",
     "description": "HTTP client made for scraping based on got.",
-    "main": "dist/index.js",
     "engines": {
-        "node": ">=15.10.0"
+        "node": ">=18"
     },
     "files": [
         "dist"
     ],
+    "exports": {
+        "types": "./dist/index.d.ts",
+        "import": "./dist/index.js"
+    },
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "type": "module",
     "dependencies": {
-        "got-cjs": "12.5.4",
-        "header-generator": "^2.1.3",
-        "http2-wrapper": "^2.1.11",
-        "mimic-response": "^3.1.0",
-        "ow": "^0.28.1",
-        "quick-lru": "^5.1.1",
-        "tslib": "^2.4.0"
+        "got": "13.0.0",
```

Comment:
i guess we dont need to pin the dependency here

Reply:
Good catch, we can use `^` for the version range